clang/lib/Sema/Sema.cpp

```diff
@@ -1792,6 +1799,7 @@ class DeferredDiagnosticsEmitter
         FirstDiag = false;
       }
     }
+    return;
```

Comment:
This `return` seems a bit redundant here.

Reply:
Yes, sorry, will remove that.